lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+
+class Menu
+  attr_reader :items
+
+  CONSTANT_ITEMS =
```

Comment:
Not sure if you need to give the array as a constant, an instance variable would be fine.